### PR TITLE
Update Privacy page

### PR DESCRIPTION
**Pull request by couci**
Small changes on language and some grammar corrections

### Description
Madee some small additions, corrected small grammar mistakes and made some points more explicit

### CODE REVIEW

File: files/en-us/web/privacy/index.md [hunk 1]
```diff
@@ -168,11 +168,11 @@ The ethics of data collection can be broken down into three simple principles:
...
 
 It is good to anonymize data. You should also consider whether you can get what you need by making your data request less granular. As an example, instead of asking a user their favorite products, you could ask them to select between more general categories.
 
-The best way to protect user privacy though is by not collecting data at all. Referring to the previous example, you could infer the same data by looking at user purchase history. As another example, users really appreciate being able to buy products anonymously. You shouldn't force them to sign up for an account; it should be their choice.
+The best way to protect user privacy though is to minimize the data you collect. Referring to the previous example, you could infer the same data by looking at user purchase history. As another example, users really appreciate being able to buy products anonymously. You shouldn't force them to sign up for an account; if it’s not necessary for the service to operate, it should be their choice.
```
Comment (sideshowbarker) [inline files/en-us/web/privacy/index.md:175]:
```suggestion
The best way to protect user privacy though is to minimize the data you collect. Referring to the previous example, you could infer the same data by looking at user purchase history. As another example, users really appreciate being able to buy products anonymously. You shouldn't force them to sign up for an account; if it's not necessary for the service to operate, it should be their choice.
```
Unfortunately, MDN policy if for some reason to not use real normal typesetting apostrophes and quotation marks, but to instead use ASCII characters.

File: files/en-us/web/privacy/index.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ Once you have decided what data you are going to collect, you should publish a p
...
 
-When you've published your privacy policy, you need to make sure that you comply with it — doing what you say you are going to do is very important in building user trust. You should only collect the data you say you'll collect, and only use it for the purpose you say you'll use it for. If someone from your company comes up with a clever new way to use existing data, that still isn't OK under the terms of your policy if it doesn't specify that you'll use it for that purpose.
+When you've published your privacy policy, you need to make sure that you comply with it — doing what you say you are going to do is very important in building user trust. You should only collect the data you say you'll collect, and only use it for the purpose you say you'll use it for. If someone from your company comes up with a clever new way to use existing data, that still isn't OK under the terms of your policy if it doesn't specify that you'll use it for that purpose. If users consented to the use of their data for a specific purpose and that purpose expands, you may have to consider obtaining new consent.
```
Comment (sideshowbarker) [inline files/en-us/web/privacy/index.md:189]:
```suggestion
When you've published your privacy policy, you need to make sure that you comply with it — doing what you say you are going to do is very important in building user trust. You should only collect the data you say you'll collect, and only use it for the purpose you say you'll use it for. If someone from your company comes up with a clever new way to use existing data, that still isn't OK under the terms of your policy if it doesn't specify that you'll use it for that purpose. If users consented to the use of their data for a specific purpose and that purpose expands, you may have to consider obtaining new consent.
```

**Comment by sideshowbarker** [comment]
(I won’t do an actual review on this one — I just stepped in to fix the CI errors from the markdown linter)

**Review by dipikabh — APPROVED** [review]
Thanks @couci - the updates look good.

**Pull request merged**